src/data/faq_de.json

```
@@ -1495,21 +1510,24 @@
                                 "anchor": "who_can_help",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Aktualisiert am ?????r</b>",
                                     "Die technische Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei technischen Fragen rund um die Corona-Warn-App, zum Beispiel bei Problemen mit der Risiko-Ermittlung. Sie erreichen sie von Montag bis Freitag von 7 bis 17 Uhr (außer an bundesweiten Feiertagen), und sie ist für Sie innerhalb Deutschlands kostenfrei.",
-                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei.",
+                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ?????h eingestellt.",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Die TAN-Hotline (<a href='#international_phone_numbers' target='_blank'>klicken Sie hier um zu den Telefonnummern zu gelangen</a>) hilft Ihnen bei der Eingabe der TAN, wenn Sie ein positives PCR-Testergebnis haben, und wenn Sie keine TAN oder kein Dokument mit PCR-Test-QR-Code erhalten haben. Sie erreichen sie von Montag bis Sonntag von 7 bis 20 Uhr Ortszeit in Deutschland. Der Anruf ist innerhalb Deutschlands kostenfrei. Beachten Sie bitte, dass ab der Version 3.0 der Corona-Warn-App keine teleTAN mehr benötigt wird, um ein positives Testergebnis zu teilen, welches nicht in die Corona-Warn-App übermittelt wurde. Daher wird der Betrieb der TAN-Hotline zum ??????h eingestellt.",
```